python-lib/dku_io_utils/partitions_handling.py

```diff
@@ -42,12 +73,13 @@ def get_dimensions(dataset):
     return dimensions, types
 
 
-def get_partitions(dku_flow_variables, dimensions):
+def get_partitions(dku_flow_variables, dimensions, source=None):
```

Comment:
Could we remove the need for `dku_flow_variables` as argument, and instead retrieve it with the Dataiku API?

Reply:
`get_partitions` and `complete_file_path_time_pattern` are both called by `get_folder_partition_root`. Retrieving `dku_flow_variables` from `get_folder_partition_root`, we can call the API once instead of twice.

Reply:
Understood, dropping it then.